src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo | null;
+  setSelectedTodo: (value: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```

Better to set `false` as a default value and change it to `true` only before you are making API call